### ISSUE TITLE
Display the code and its resulting PDF side by side

### ISSUE THREAD

**Issue by dbitouze** [davidcarlisle/latexcgi#10]
It would be nice to have an option letting us display the codes and their resulting PDFs, not the latter below the former, but side by side as it is commonly the case in (La)TeX editors. One advantage is to more easily look at:

- the code corresponding to what is seen on the PDF,
- the PDF corresponding to what is seen on the code.

**Comment by davidcarlisle** [comment]
yes I have code for that I sent a variant version when someone asked last year.  I may try to merge it back as an option, only changes a few lines but in separate blocks affecting css and js bits so not immediately obvious the best way to parameterise a merged version, but it's less than 10 lines changed so how hard can it be....

See https://ktychan.gitlab.io/teaching/1200-latex/ for a live example
